src/index.js

```diff
@@ -97,7 +97,7 @@ const getUsers = async () => {
       const data = await response.json();
       const users = data.result;
 
-      ul.innerHTM = '';
+      ul.innerHTML = '';
```

Comment:
- [ ] Your design looks awesome, but we have a small thing to fix before, See, you have imported the spinner.gif as Spinner at the top of the file, but when calling the variable, you did it as a string on line 91 as shown in the image below
<img width="914" alt="3" src="https://github.com/evansnyamekye/Leaderboard/assets/77843050/3910ce50-9bb4-4f67-8b78-13574b163f90">

- [ ] This makes your loader crash on the browser because the image is not recognized
<img width="218" alt="1" src="https://github.com/evansnyamekye/Leaderboard/assets/77843050/dd2d50eb-0c97-4e25-89a1-b5eef6d23b97">
but after I removed the single quote from the Spinner on line 91, we got this, as simple as it can appear, please make suer you follow my example and resubmit your work after the bug was fixed
<img width="208" alt="2" src="https://github.com/evansnyamekye/Leaderboard/assets/77843050/ce385029-5902-4507-9695-b2ce4f6e7794">

Reply:
### @joskalenda well noted. Will fix the problem as requested. 

Thank you 🛩️.